homeassistant/components/homekit/strings.json

```diff
@@ -1,25 +1,26 @@
 {
-    "title": "HomeKit Bridge",
+    "title": "HomeKit",
```

Comment:
We can drop this, it will automatically use the one from `manifest.json`. And then we should update `manifest.json` too.
```suggestion
```